crates/ruff_python_parser/tests/snapshots/invalid_syntax@try_stmt_misspelled_except.py.snap

```diff
@@ -198,8 +197,8 @@ Module(
   |
 3 | exept:  # spellchecker:disable-line
 4 |     pass
+  |         ^ Syntax Error: Expected a statement
```

Comment:
Hmm. I guess this is another case where the offset used to be at the start of the line and the caret now ends up at the end of the previous line. Unfortunately, this will be harder to fix, and I don't think we have tests for every position where we expect a node.

Reply:
Aye, I filed an issue about this: https://github.com/astral-sh/ruff/issues/15510